src/bundle/Resources/config/services/menu.yaml

```diff
@@ -174,6 +181,11 @@ services:
         tags:
             - { name: kernel.event_listener, event: ezplatform_admin_ui.menu_configure.main_menu, method: moveAdminToLast, priority: -50 }
 
+    # regroup menu listener
```

Comment:
```suggestion
```

Reply:
No need for this class in reorganized code